.github/workflows/blank.yml

```diff
@@ -33,6 +33,7 @@ jobs:
           ./gradlew -Pskiko.native.enabled=true macosX64Test iosX64Test
           ./gradlew -Pskiko.native.enabled=true publishToMavenLocal
       - uses: actions/upload-artifact@v2
+        if: ${{ true }}
```

Comment:
Maybe remove those.

Reply:
that was an attempt to make `upload-test-results` step run despite failing tests. But `${{true}}` didn't help for some reason. By default a step gets executed when preceding steps were successful. This issue doesn't let us download the test report when any test failed - not very useful. I'll check this problem later.

Removed for now.